framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Button/index.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as React from 'react';
+import styles from './Button.module.scss';
+import Text from '../Text';
+
+interface Props {
+	name: string;
```

Comment:
The issue didn't mentioned to have a `name` prop to use as button text. Rather as normal React component `Button` should just render the `children` prop.